Sell/Order/Models/Order.swift

```diff
@@ -87,7 +87,11 @@ extension Order {
         createdAt = response.createdAt?.iso8601Date
         estimatedAt = response.estimatedAt?.iso8601Date
         collectionPreferenceType = CollectionPreferenceType(apolloType: response.collectionPreferenceType)
-        seatInfo = SeatInfo(response: response.seatInfo?.fragments.fragmentSeatInfo)
+        if let responseSeatInfo = response.seatInfo {
```

Comment:
If we can't auto-decode, is there a way to not duplicate the initialisation of the seatInfo across anywhere it needs to be decoded?
Could SeatInfo(jsonValue value: JSONValue) become SeatInfo(jsonValue value: JSONValue?) to avoid the if let requirement?

Reply:
As it needs to be conform to JSONDecodable, I cannot change the interface `public init(jsonValue value: JSONValue) throws`. But I created another init function to accept optional JSON for SeatInfo, so that we can avoid the if let logic. Thank you.